examples/notebooks/load-modelling.ipynb

```diff
@@ -0,0 +1,508 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Add load shedding as generator\n",
+    "network.add(\n",
+    "    \"Generator\",\n",
+    "    \"load shedding\",\n",
+    "    bus=\"My bus\",\n",
+    "    p_nom=50,\n",
+    "    carrier=\"load\",\n",
+    "    p_max_pu=1.0,\n",
+    "    marginal_cost=1e2,  # 100 €/MWh\n",
+    "    p_nom_extendable=True,\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After adding load shedding, the optimization will pass as the nodal energy balance is fulfilled"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "network.optimize()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Looking at the optimization results, the mismatch is exactly compensated by load shedding."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots()\n",
+    "\n",
+    "network.generators_t.p.plot(ax=ax)\n",
+    "network.loads_t.p.plot(ax=ax)\n",
+    "\n",
+    "ax.set_ylabel(\"Power (MW)\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load shifting"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\"Load shifting\" is often used to provide network benefits arising from demand side flexibility. A classic application in the electricity domain is **Electric Vehicle (EV)** battery. EVs can be scheduled to charge during off-peak hours when electricity demand is low, and electricity is cheaper. Furthermore, with Vehicle-to-Grid (V2G) capabilities, EVs can potentially supply power back to the grid during peak demand. In our model, we'll represent EV charging and potential discharging as storage units, which effectively act as energy time-shifters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# add EV battery as storage unit\n",
+    "network.add(\n",
+    "    \"StorageUnit\",\n",
+    "    \"EV battery\",\n",
+    "    bus=\"My bus\",\n",
+    "    p_nom=1,  # positive to indicate that there is a devise\n",
+    "    carrier=\"electricity\",\n",
+    "    p_nom_extendable=True,\n",
+    "    standing_loss=0.01,\n",
+    "    cyclic_stage_of_charge=True,\n",
```

Comment:
A typo should by fixed: `cyclic_stage_of_charge ` -> `cyclic_state_of_charge`

Reply:
fixed ✔️